src/history.rs

```diff
@@ -53,23 +54,25 @@ impl History {
         })
     }
 
-    pub fn undo(&mut self, rows: &mut Vec<Row>) -> Option<(usize, usize, usize)> {
-        self.finish_ongoing_edit();
+    pub fn undo(&mut self, rows: &mut Vec<Row>) -> Option<(usize, usize, usize, bool)> {
+        let ongoing = self.finish_ongoing_edit();
```

Comment:
I feel this variable name `ongoing` is not suitable. It represents wether some diff was applied or not. So I feel `edited` or `modified` is more suitable.

Reply:
Renamed to `edited`.